testnet/stacks-node/src/burnchains/bitcoin_regtest_controller.rs

```diff
@@ -721,7 +851,15 @@ impl BitcoinRegtestController {
         tx.output = vec![consensus_output];
         tx.output.push(payload.output.to_bitcoin_tx_out(output_amt));
 
-        self.finalize_tx(&mut tx, output_amt, DUST_UTXO_LIMIT, utxos, signer, 1)?;
+        self.finalize_tx(
+            &mut tx,
+            output_amt,
+            0,
+            280,
```

Comment:
What's `280` here?

Reply:
`280` is the first estimated size we're using for the bitcoin `TransferStx`  operations (it's a `#[cfg(test)]` method and used once, but I can use a constant if you prefer)

Reply:
Refactored in https://github.com/blockstack/stacks-blockchain/pull/2383/commits/4d576c9ffe82b31c8efe4b4659f30b6fa7e576cd